book/01-guided-tour.html

```diff
@@ -177,22 +189,47 @@ <h1>Functions and Type Inference</h1>
       <code>int -&gt; int</code> is a function type, in this case
       indicating a function that takes an <code>int</code> and
       returns an <code>int</code>. We can also write functions that
-      take multiple arguments. (Note that the following example
-      will not work if you haven't opened
-      <code>Core_kernel</code> as was suggested
-      earlier.)<idx>multi-argument
-      functions</idx><idx>functions/with multiple
-      arguments</idx></p>
+      take multiple arguments. (Reminder: Don't forget to open <code>Base</code>,
+      or these examples won't work!)
+
+      <idx>multi-argument functions</idx>
+      <idx>functions/with multiple arguments</idx>
+      </p>
+
       <link rel="import" href="code/guided-tour/main.topscript" part="6"/>
 
-      <p>The preceding example also happens to be our first use of
-      modules. Here, <code>Float.of_int</code> refers to the
-      <code>of_int</code> function contained in the
-      <code>Float</code> module. This is different from what you
-      might expect from an object-oriented language, where
-      dot-notation is typically used for accessing a method of an
-      object. Note that module names always start with a capital
-      letter.</p>
+      <p>
+	The preceding example also happens to be our first use of
+	modules. Here, <code>Float.of_int</code> refers to
+	the <code>of_int</code> function contained in
+	the <code>Float</code> module. This is different from what you
+	might expect from an object-oriented language, where
+	dot-notation is typically used for accessing a method of an
+	object. Note that module names always start with a capital
+	letter.
+      </p>
+
+      <p>
+	Modules can also be opened, as we opened <code>Base</code>
```

Comment:
"Modules can also be opened" doesn't actually tell us what happened as a result of executing the `open Base`.  Perhaps note that all of the values contained within the module are now available in the toplevel environment?

Reply:
I expanded the language there.